tests/Integration/EmptyArraysHandlingTest.php

```diff
@@ -22,9 +23,11 @@ public function testEmptyArrayAsListIsHandled()
         MERGE (f)-[:KNOWS]->(n)';
 
         $params = ['id' => 'me', 'friends' => Collections::asList([])];
-        $this->getSession()->run($query, $params);
 
-        $result = $this->getSession()->run('MATCH (n:User) RETURN count(n) AS c');
+        $session = $this->getSession();
```

Comment:
This goes in contradiction to https://github.com/graphaware/neo4j-bolt-php/pull/24/files#diff-0c380b4ddfb5d56023500d4614d375bdR44

Reply:
In this case, the session is used multiple times when the file you're referring to it's not the case ;)